src/Extensions/Logging/InitialisationLogger.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+using System.Diagnostics;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Omex.Extensions.Logging
+{
+	/// <summary>
+	/// InitStaticLogger is the logger to be used before the proper ILogger from DI is set
+	/// not to be used as main logger
+	/// </summary>
+	public static class InitialisationLogger
+	{
+		/// <summary>
+		/// Instance of logger
+		/// </summary>
+		public static ILogger Instance { get; private set; } = LoggerFactory.Create(builder =>
+		 {
+			 builder.LoadInitialisationLogger();
+		 }).CreateLogger("Initial-Logging");
+
+		/// <summary>
+		/// Log on success
+		/// </summary>
+		/// <param name="serviceNameForLogging">Service name for logging</param>
+		/// <param name="message">Message to log</param>
+		public static void InitilisationSucceed(string serviceNameForLogging, string message = "")
+		{
+			string logMessage = string.IsNullOrWhiteSpace(message) ?
+				$"Initilisation logger logging success to {serviceNameForLogging}" : message ;
```

Comment:
In my mind success what about successful initialization of the service(or host), not a logger, maybe we could adjust the message into something like "Initialization successful for {serviceNameForLogging}. {message}"